packages/valory/skills/abstract_round_abci/test_tools/integration.py

```diff
@@ -284,7 +284,7 @@ def process_n_messages(
         return tuple(incoming_messages)
 
 
-class HardHatHelperIntegration(IntegrationBaseCase):
+class HardHatHelperIntegration(IntegrationBaseCase):  # pragma: no cover
```

Comment:
this is not used anywhere in this repository

https://github.com/valory-xyz/open-autonomy/blob/16041619aa63ecad6beec6cc2e4f872d2223748a/packages/valory/skills/abstract_round_abci/test_tools/integration.py#L287-L300